Makefile

```diff
@@ -289,6 +289,7 @@ create-management-cluster: $(KUSTOMIZE) $(ENVSUBST) $(KUBECTL) $(KIND) ## Create
 	$(KUBECTL) wait --for=condition=Available --timeout=5m -n capi-system deployment -l cluster.x-k8s.io/provider=cluster-api
 	$(KUBECTL) wait --for=condition=Available --timeout=5m -n capi-kubeadm-bootstrap-system deployment -l cluster.x-k8s.io/provider=bootstrap-kubeadm
 	$(KUBECTL) wait --for=condition=Available --timeout=5m -n capi-kubeadm-control-plane-system deployment -l cluster.x-k8s.io/provider=control-plane-kubeadm
+	timeout --foreground 300 bash -c "until $(KUBECTL) get clusterresourcesets -A; do sleep 3; done"
```

Comment:
In the logs this printed:

```shell
timeout --foreground 300 bash -c "until /home/prow/go/src/sigs.k8s.io/cluster-api-provider-azure/hack/tools/bin/kubectl-v1.25.6 get clusterresourcesets -A; do sleep 3; done"
No resources found
# install Windows Calico cluster resource set
```

Shouldn't the script return an error if no CRSs are found?

Reply:
Good question!

No, it should return an error if the ClusterResourceSet CRD hasn't yet been installed. While I was putting this together I noticed that flake (we install the calico CRS and the cluster complains about not knowing about that resource type).